packages/expo-notifications/android/src/main/AndroidManifest.xml

```diff
@@ -7,7 +7,7 @@
         <service
             android:name=".FirebaseListenerService"
             android:exported="false">
-            <intent-filter>
+            <intent-filter android:priority="-1">
```

Comment:
Could you add a _bug fix_ entry to CHANGELOG that you fixed the ability to override the `FirebaseListenerService` without having to add a custom `priority`? Let's be super explicit about our changes and progress! 👍